package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-pages": "^0.3.4",
```

Comment:
I'm realising although we have concluded that we will be using react packages, because we haven't implemented the package on this main.jsx component, we didn't really need to add this load on the project by installing this package right now. Ann error in design, and acceptance criteria for this component. But its all good. We will merge for now, but I think we should look out for such details in the future so that we try building small pieces of code that we need at the moment.